schema-bridge.graphql

```diff
@@ -426,11 +758,271 @@ type PoolHourlySnapshot @entity {
   rewardTokenEmissionsUSD: [BigDecimal!]
 }
 
+type PoolRouteSnapshot @entity {
```

Comment:
what is the time interval on this snapshot?

Reply:
Depends on its parent. If it is inside `PoolHourlySnapshot.routes` it will be hourly. Same with the daily ones.

I made it this way because since they are a sub-entity of the PoolSnapshot they won’t be queried directly, they will always be retrieved as part of the pool snapshot.